layout/common/article.ejs

```diff
@@ -8,8 +8,10 @@
         <% if (post.layout != 'page') { %>
             <div class="article-meta">
                 <%- partial('post/date', { class_name: 'article-date', date_format: null }) %>
-		<%- partial('post/author') %>
+		        <%- partial('post/author') %>
```

Comment:
tabs vs spaces... replace the tab with the correct amount of spaces.

Reply:
OK, already replace all tab with spaces in commits on Oct 9, 2019.